Dockerfile

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
This is a library. Why is there a `Dockerfile` for it?

Reply:
@hankwallace, when publishing lform on npm we need to build lform, and lform requires a node 10 version, but in CICD jobs we cannot be able to change the node version. Therefore, I use a Docker file for building the form

Reply:
That makes sense. It's unfortunate that we have to add the `Dockerfile` here though. 

@semicolin @ddjain can you think of a better way?

Reply:
Maybe a GitHub workflow?